Tools/MGCB/BuildContent.cs

```diff
@@ -284,6 +284,20 @@ public void Build(out int successCount, out int errorCount)
             errorCount = 0;
             successCount = 0;
 
+            // Before building the content, register all files to be built. (Necessary to
+            // correctly resolve external references.)
+            foreach (var c in _content)
```

Comment:
I think the source files should be processed in the original order, i.e. the order of the commands in the .mgcb file. 

The order of the content affects the name resolution for external references (TextureC.png -> TextureC_0.xnb, TextureC_1.xnb). The build order should be consistent between builds. When a new content is added, it is appended at the end of the .mgcb files and built last. Content that is already built is not affected (partial build).
However, when we sort the content, adding a new content changes the build order, which may change the names for external references (TextureC_0.xnb <-> TextureC_1.xnb). This *could* be a problem for partial builds. I'm not sure. I recommend leaving as is.